libsolidity/interface/CompilerStack.cpp

```diff
@@ -1245,6 +1246,27 @@ void CompilerStack::storeContractDefinitions()
 			}
 }
 
+void CompilerStack::annotateInternalFunctionIDs()
+{
+	for (Source const* source: m_sourceOrder)
+	{
+		if (!source->ast)
+			continue;
+
+		for (ContractDefinition const* contract: ASTNode::filteredNodes<ContractDefinition>(source->ast->nodes()))
+		{
+			ContractDefinitionAnnotation& annotation =
+				m_contracts.at(contract->fullyQualifiedName()).contract->annotation();
```

Comment:
Why all the indirection? Isn't this just the very same as plainly
```suggestion
				contract->annotation();
```
?

Reply:
Yes, it is. Rookie mistake :)